setup.py

```diff
@@ -13,7 +13,8 @@
     install_requires=[
         'requests>=2.26.0',
         'jsonrpcclient==3.3.6',
-        'retry==0.9.2'
+        'retry==0.9.2',
+        'bump2version'
```

Comment:
This seems redundant. You already have `python -m pip install bump2version`. This would mean that if you do `pip install hm-pyhelper` you will also get `bump2version` installed, which is unnecessary.

Reply:
> This seems redundant. You already have `python -m pip install bump2version`. This would mean that if you do `pip install hm-pyhelper` you will also get `bump2version` installed, which is unnecessary.

I'll remove this from the code, thank you for catching that. I think I added it in at the start as I saw the requires. I'll do the push after I add in the user/email setting in the git action